modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/search_paginator.jsp

```diff
@@ -0,0 +1,15 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<clay:container-fluid>
+	<liferay-ui:search-paginator
+		markupView="lexicon"
+		searchContainer="<%= searchPaginatorDisplayContext.getSearchContainer() %>"
+	/>
+</clay:container-fluid>
```

Comment:
@dsanz I see what you mean, this is not right. 

Do we maybe want a sample module for all tags? Something like `frontend-taglib/frontend-taglib-sample-web`. In it, we could have several pages:
- Clay
- Liferay UI
- Liferay Frontend
- AUI
- etc

Within these, we would need to fit a lot of tags. These could be one subpage per tag, maybe accessed with vertical navigation.

In PW, they can all be under `frontend-taglib` project, further split into `/tests/clay`, `/tests/liferay-ui,` etc.

What do you think?

Reply:
The pw side is more manageable as we can mimic the exact path. I'm struggling more with the sample-web module in DXP. 

`frontend-taglib/frontend-taglib-sample-web` sounds reasonable to me, as a sort of "bag" to include any portal web tag lib sample

Then in tests we would use that same folder as you suggest, with a separate forlder per tag namespace, except for clay, which has its own sample and so it can enjoy its own folder inside pw/tests

Does that sound right to you?

Reply:
Yeah, sounds good. I included Clay for completion sake, as I don't see any reason for it not to be in the bag as well. But migrating it can be a very low priority.